recipe/meta.yaml

```diff
@@ -1,35 +1,33 @@
-{% set name = "pytest-notebook" %}
-{% set version = "0.6.1" %}
+{% set name = "pytest_notebook" %}
```

Comment:
IIRC on conda-forge dash is more used than underscore.
so probably if we want to have a name with underscore we should create a new output to have both with dash and with underscore.

the same approach used by pre-commit hook https://github.com/conda-forge/pre-commit-feedstock/blob/main/recipe/meta.yaml#L53